### DISCUSSION TITLE
Owner and group error on web-based file manager

### DISCUSSION THREAD

**Discussion by tizfra** [GeekFunkLabs/fluidpatcher#57] [Q&A] [answered]
Hi,
I've found that when I copy or move file on web-based file manager it changes owner and group of that file so I'm not able to modify that file on squishbox.
I need to use `chown` command on putty to set correct owner and user.

I've another question: how can I delete character on bank's name on squishbox?

Thank you.

**Comment by albedozero** [comment, marked as answer]
This could be because you are running an older version of tinyfilemanager, or it may have been set up improperly by an older version of the squishbox install script. You should be able to fix this by re-running the install script (`curl -L git.io/squishbox | bash`) and responding `n` or `0` (no change) to all the prompts except the one about setting up the web-based file manager. That should install the latest version and set up the correct groups/permissions. If that doesn't work, let me know.

On the second question: I didn't provide a menu option to rename or delete banks on the squishbox - I was trying to keep the number of menu items down and wanted to save users from accidentally deleting entire banks. You can use the 'Save Bank' option on squishbox to save a copy of the bank with the one character removed, but to rename/delete banks you have to use the web file manager or log in remotely.
